### DISCUSSION TITLE
求教新版的nb create后无之前的bot src 等目录 该如何配置相关文件呢，求教程

### DISCUSSION THREAD

**Discussion by mrchenxxx** [nonebot/discussions#318] [问答] [answered]
只有一个pyproject.toml 我不知道咋配置啊

**Comment by yanyongyu** [comment, marked as answer]
你创建项目的时候选择simple模板就有src目录了，新的文档施工中，你可以在https://deploy-preview-1604--nonebot2.netlify.app/docs/next/ 查看
